### DISCUSSION TITLE
Grafana Metrics Download/Upload Throughput not working

### DISCUSSION THREAD

**Discussion by ShubhamTatvamasi** [magma/magma#8187] [Q&A]
Hi guys, I am not getting metrics for Download Throughput and Upload Throughput. Any idea where to look for a fix?

![Screenshot from 2021-07-20 15-28-00](https://user-images.githubusercontent.com/14152150/126368064-abbd50de-75e7-47f1-8578-7b3f75655675.png)

**Comment by ekowtaylor** [comment]
cc. @andreilee

**Comment by ShubhamTatvamasi** [comment]
Are these volumes correct?

![Screenshot from 2021-07-21 11-24-26](https://user-images.githubusercontent.com/14152150/126438319-6fae53bc-c532-40c2-b3d4-fde7098074e0.png)

**Comment by ShubhamTatvamasi** [comment]
The default version is 6.6.2, but I have upgraded it to 6.7.6, it's still not working.
